test/test_bin_bool_ops.py

```diff
@@ -66,5 +67,30 @@ def test_combo_nested(self):
         py_code = '1 or 2 and 3 or 4 + 3'
         self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
         self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+
+    def test_compare_greater_than(self):
+        js_code = '1 > 2'
+        py_code = '1 > 2'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+
+    def test_compare_less_than_or_equal(self):
+        js_code = '1 <= 2'
+        py_code = '1 <= 2'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+        
+    def test_compare_equal(self):
+        js_code = '1 == 2'
+        py_code = '1 == 2'
+        self.assertEqual(py_code, main.main(js_code, 'js', 'py'))
+        self.assertEqual(js_code, main.main(py_code, 'py', 'js'))
+
+    def test_comparemulti_equal(self):
+        js_code = '1 == 2 == 3'
+        py_code = '1 == 2 == 3'
```

Comment:
Given this seems like the most delicate part of the python implementation, would be great to have a few more variations of more than one operators.

Reply:
Good idea- just did it!